tests/topology/input/bgp-unnumbered-dual-stack.yml

```diff
@@ -53,4 +53,3 @@ links:
   prefix:
     ipv4: True
     ipv6: 2001:db8:1::/64
-
```

Comment:
This last test should be "IPv4+6 AF over IPv6 numbered, no IPv4 session"

Reply:
Correct. That's why I'm using **none** device ;)